internal/operators/mce/manifest.go

```diff
@@ -2,11 +2,14 @@ package mce
 
 import (
 	"bytes"
+	"fmt"
 	"text/template"
+
+	"github.com/go-openapi/swag"
 )
 
 // Manifests returns manifests needed to deploy MCE.
-func Manifests() (openshiftManifests map[string][]byte, customManifests []byte, err error) {
+func Manifests(openshiftVersion string) (openshiftManifests map[string][]byte, customManifests []byte, err error) {
```

Comment:
not your doing but, this function has an  return param `err` but it never return's an error

Reply:
This is an interface function

Reply:
so lets remove the return err

```suggestion
func Manifests(openshiftVersion string) (openshiftManifests map[string][]byte, customManifests []byte) {
```

Reply:
```
	namespaceManifest, err := getNamespace()
	if err != nil {
		return
	}
	operatorGroupManifest, err := getOperatorGroup()
	if err != nil {
		return
	}
	operatorSubscriptionManifest, err := getSubscription(openshiftVersion)
	if err != nil {
		return
	}
```